### PR TITLE
Update Refusal Logic

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
New changes to refusal types. Need to be able to change refusal types and stop reminders/visits where necessary 

# What has changed
Column type changed from boolean to string

# How to test?
Build image with rest of repos on card and run ATs

# Links
https://trello.com/c/tnVfjzpv/710-update-refusal-logic-8

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Update Refusal logic (8)](https://trello.com/c/tnVfjzpv/710-update-refusal-logic-8)

**Review by LukeLoze1 — APPROVED** [review]

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java [hunk 1]
```diff
@@ -119,8 +119,7 @@
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean receiptReceived;
 
-  @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
-  private boolean refusalReceived;
+  @Column private String refusalReceived;
```
Comment (NickGrantX) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java]: Should use enum

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Needs to use enum to meet coding standards

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**